src/core/github.js

```diff
@@ -9,6 +9,44 @@ import { pub } from "core/pubsubhub";
 
 export const name = "core/github";
 
+var body = document.body;
+var head = document.head;
+var issueBase;
+
+var fileIssueLink = document.createElement('button');
+fileIssueLink.setAttribute('data-balloon','file an issue related to the text selected');
+fileIssueLink.setAttribute('data-balloon-pos','left');
+fileIssueLink.classList.add('removeOnSave');
+fileIssueLink.id = 'respec-file-issue-btn';
+fileIssueLink.innerHTML = 'File Issue';
+fileIssueLink.onclick = function () {
+  window.location.href = fileIssueLink.getAttribute('data-href');
+  fileIssueLink.style.display = 'none';
+};
+
+var link = document.createElement('link');
+link.rel = 'stylesheet';
+link.href = "https://cdnjs.cloudflare.com/ajax/libs/balloon-css/0.5.0/balloon.min.css";
```

Comment:
Generally, we want control our own dependencies. So, we want to:

1. `npm install balloon-css` 
1.  add the patch to [tools/copydeps.js](https://github.com/w3c/respec/blob/develop/tools/copydeps.js#L13)
1. import the css as text, and add it to this `link` (see how we do that in "w3c/styles").

Reply:
Ok

Reply:
fixed typo above... should have been "path" not "patch".

Reply:
ok